.nais/job-trigger-prod.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     team: team-esyfo
 spec:
   image: {{ image }}
-  schedule: "05 09 * * *"
+  schedule: "05 09,18 * * *"
```

Comment:
mangler det et mellomrom her?

Reply:
Tror ikke det. 
Testet i dev [denne commiten](https://github.com/navikt/esyfovarsel/pull/277/commits/f8fc00136c5e11ad09963f8a5982abdd4d8ace57), så ble jobben kjørt 2 ganger [som forventet](https://logs.adeo.no/app/discover#/view/4ce734f0-9646-11eb-a3f3-5be089f1b0b9?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-03-22T12:00:00.000Z',to:'2023-03-22T16:00:00.000Z'))&_a=(columns:!(message,envclass,level,application,host),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:envclass,negate:!f,params:(query:q),type:phrase),query:(match_phrase:(envclass:q))),('$state':(store:appState),meta:(alias:eSYFO-applikasjoner,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:application,negate:!f,params:!(oppfolgingsplan,dialogmotearbeidsgiver,syfobrukertilgang,syfomotebehov,syfooppdfgen,syfooppfolgingsplanservice,dialogmote-frontend,esyfovarsel-job,esyfovarsel-planlegging-job,esyfo-info-frontend,esyfovarsel),type:phrases,value:!(oppfolgingsplan,dialogmotearbeidsgiver,syfobrukertilgang,syfomotebehov,syfooppdfgen,syfooppfolgingsplanservice,dialogmote-frontend,esyfovarsel-job,esyfovarsel-planlegging-job,esyfo-info-frontend,esyfovarsel)),query:(bool:(minimum_should_match:1,should:!((match_phrase:(application:oppfolgingsplan)),(match_phrase:(application:dialogmotearbeidsgiver)),(match_phrase:(application:syfobrukertilgang)),(match_phrase:(application:syfomotebehov)),(match_phrase:(application:syfooppdfgen)),(match_phrase:(application:syfooppfolgingsplanservice)),(match_phrase:(application:dialogmote-frontend)),(match_phrase:(application:esyfovarsel-job)),(match_phrase:(application:esyfovarsel-planlegging-job)),(match_phrase:(application:esyfo-info-frontend)),(match_phrase:(application:esyfovarsel)))))),('$state':(store:appState),meta:(alias:!n,disabled:!t,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:level,negate:!f,params:(query:Error),type:phrase),query:(match_phrase:(level:Error)))),grid:(),hideChart:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',interval:auto,query:(language:lucene,query:'%22Starter%20SendVarslerJobb%22'),sort:!(!('@timestamp',desc)))).
Mellomrom blir tolket som felt separator.